packages/core/src/common/_color-aliases.scss

```diff
@@ -189,41 +189,41 @@ consistent color usage across components.
 Styleguide variables.colors
 */
 
-$pt-intent-primary: $blue3;
-$pt-intent-success: $green3;
-$pt-intent-warning: $orange3;
-$pt-intent-danger: $red3;
+$pt-intent-primary: $blue3 !default;
+$pt-intent-success: $green3 !default;
+$pt-intent-warning: $orange3 !default;
+$pt-intent-danger: $red3 !default;
 
-$pt-app-background-color: $light-gray5;
-$pt-dark-app-background-color: $dark-gray3;
+$pt-app-background-color: $light-gray5 !default;
+$pt-dark-app-background-color: $dark-gray3 !default;
 
-$pt-text-color: $dark-gray1;
-$pt-text-color-muted: $gray1;
-$pt-text-color-disabled: rgba($pt-text-color-muted, 0.5);
-$pt-heading-color: $pt-text-color;
-$pt-link-color: $blue2;
-$pt-dark-text-color: $light-gray5;
-$pt-dark-text-color-muted: $gray5;
-$pt-dark-text-color-disabled: rgba($pt-dark-text-color-muted, 0.5);
-$pt-dark-heading-color: $pt-dark-text-color;
-$pt-dark-link-color: $blue4;
+$pt-text-color: $dark-gray1 !default;
+$pt-text-color-muted: $gray1 !default;
+$pt-text-color-disabled: rgba($pt-text-color-muted, .5) !default;
+$pt-heading-color: $pt-text-color !default;
+$pt-link-color: $blue2 !default;
+$pt-dark-text-color: $light-gray5 !default;
+$pt-dark-text-color-muted: $gray5 !default;
+$pt-dark-text-color-disabled: rgba($pt-dark-text-color-muted, .5) !default;
+$pt-dark-heading-color: $pt-dark-text-color !default;
+$pt-dark-link-color: $blue4 !default;
 // Default text selection color using #7dbcff
-$pt-text-selection-color: rgba(125, 188, 255, 0.6);
+$pt-text-selection-color: rgba(125, 188, 255, .6) !default;
```

Comment:
Decimals start with `0` for clarity, I think the linter will give you trouble with this change